packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -612,6 +575,7 @@ const BlocksContent = ({ placeholder }: BlocksInputProps) => {
       paddingBottom={3}
     >
       <StyledEditable
+        aria-labelledby={ariaLabel}
```

Comment:
Is this to associate the label to the editing field? If so, do you not think using the `id` `htmlFor` pattern is better?

Reply:
Editable is not an `input` element but div with `role=textbox` so `id` `htmlFor` isnt supported, I tried it.

Reply:
oh that's cool, TIL.